### PR TITLE
Nickytonline/changed grouping of components in storybook

### PR DESCRIPTION
**Pull request by nickytonline**
## What type of PR is this? (check all applicable)

- [x] Refactor
- [ ] Feature
- [ ] Bug Fix
- [ ] Optimization
- [ ] Documentation Update

## Description

Changed how Application Components and Crayons (design system) are organized in Storybook. Based off of @ludwiczakpawel's [comment](https://github.com/thepracticaldev/dev.to/pull/6578#discussion_r390894608) in #6578

Latest [Storybook available](https://devto-storybook.now.sh) for your viewing pleasure.

Note: It's grouped as `Components/HTML/*` because we will eventually have Preact components as well. This is just the beginning of the design system integration into Storybook.

## Mobile & Desktop Screenshots/Recordings (if there are UI changes)

image

## Added tests?

- [ ] yes
- [x] no, because they aren't needed
- [ ] no, because I need help

## Added to documentation?

- [ ] docs.dev.to
- [ ] readme
- [x] no documentation needed

## [optional] What gif best describes this PR or how it makes you feel?

Girl using magic powers to organize her doll house

### CODE REVIEW

File: app/javascript/designSystem/__stories__/typography.scss [hunk 1]
```diff
@@ -0,0 +1,6 @@
+p::after {
```
Comment (nickytonline) [inline app/javascript/designSystem/__stories__/typography.scss]: This just shows in the typography examples the classes used to generate that particular piece of text, e.g.

image

I thought it was a good idea, but welcome to feedback @ludwiczakpawel. Also, I just randomly picked orange for the color.

Comment (ludwiczakpawel) [reply app/javascript/designSystem/__stories__/typography.scss]: sooo soo good! <3

**Comment by ludwiczakpawel** [comment]
I'm being nitpicky here, but can we have Base above the Components, not below?

**Comment by nickytonline** [comment]
@ludwiczakpawel, I want that too, but I believe it requires a Storybook upgrade which will follow the frontend infrastructure upgrades. Next week I’m going to focus on those infra issues.

**Comment by ludwiczakpawel** [comment]
Cool! Other than that it's looking great!

**Pull request merged**